regen/feature.pl

```diff
@@ -544,7 +544,7 @@ =head1 AVAILABLE FEATURES
 
 =head2 The 'say' feature
 
-C<use feature 'say'> tells the compiler to enable the Perl 6 style
+C<use feature 'say'> tells the compiler to enable the Raku style
```

Comment:
I kind of feel like both of these would better read "Raku-inspired" or similar at this point

Reply:
Resolved in last push.